src/Extensions/Logging/Internal/EventSource/EmptyServiceContext.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+
+namespace Microsoft.Omex.Extensions.Logging
+{
+	/// <summary>IServiceContext without any information</summary>
```

Comment:
`<summary>` tags on their own lines as per convention

Reply:
Fixed, thank you